10up/distributor
# Toggle Distributor controls when post status changes

### Description of the Change

Renders the Distributor element in the admin bar whether the post status is distributable or not. If it's not, the element is hidden by CSS. When the post status becomes distributable, the element is displayed.

Also adds a `watch` command to the `npm` scripts.

Closes #141 

### How to test the Change
1. Edit a Draft post. Note that the Distributor button in the admin bar is not available, and the Distributor element in the Inspector says "Distribution options available once published".
2. Publish the post. Note that the Distributor button in the admin bar is now available, and the Distributor element has a button to "Distribute page".
3. Reload the editor page. Note that the Distributor button in the admin bar is still available, as is the "Distribute page" button.
4. Click `Switch to draft` and unpublish the post. Note that the admin bar and Distributor Inspector element are as they were in step 1.

### Changelog Entry
> Changed - Toggles the Distributor admin bar element in Gutenberg based on the post status.

### Credits

Props @ggutenberg 

### Checklist:
